pan/types.pan

```diff
@@ -837,24 +851,25 @@ function is_network_name = {
         error ("usage: is_network_name (string)");
     };
 
-    if (is_hostname (ARGV[0])) {
-        return (true);
+    if (is_hostname(ARGV[0])) return(true);
+
+    if (is_shorthostname(ARGV[0])) {
+        deprecate(0, "Short hostnames are deprecated as valid network_names.");
+        return(true);
     };
 
+    if (is_top_level_domain(ARGV[0])) return(true);
+
     # Not a hostname. Is it a IP/mask?
```

Comment:
i meant this block of code `is_ip_and_mask`? (i know it's not your code and all that, so feel free to ignore)